android/Intrepid.xml

```diff
@@ -247,4 +255,7 @@
       </rules>
     </arrangement>
   </codeStyleSettings>
-</code_scheme>
+  <codeStyleSettings language="kotlin">
+    <option name="CODE_STYLE_DEFAULTS" value="KOTLIN_OFFICIAL" />
+  </codeStyleSettings>
+</code_scheme>
```

Comment:
Add newline at bottom of file?

Reply:
This is auto-generated by the IDE so I don't think it would stick if i did that.